win32ss/gdi/ntgdi/font.h

```diff
@@ -18,7 +18,7 @@ typedef struct _FONT_ENTRY_MEM
 typedef struct _FONT_ENTRY_COLL_MEM
 {
     LIST_ENTRY ListEntry;
-    UINT Handle;
+    UINT_PTR Handle;
```

Comment:
Any idea why this isn't a HANDLE in the first place?

Reply:
Because it's actually just an index, specifically:
`EntryCollection->Handle = ++Win32Process->PrivateMemFontHandleCount;`

Reply:
Well, a handle can be built any way you want, it's still a handle, ie an opaque value to the API callers. Next uses seem to indicate that it should be a handle.